### DISCUSSION TITLE
Sklearn and Xarray

### DISCUSSION THREAD

**Discussion by nagrawa6** [pydata/xarray#7251] [Q&A]
Hello!

I am trying to use Xarray data (time, ydim, xdim) into an sklearn linear regression model. However, Python is not able to perform using my 3d data. Any suggestions?

**Comment by raybellwaves** [comment]
What’s sizes are you working with?

doing .to_dataframe() is the easiest.

there  are a couple of packages for parallel fits on each grid cell for example

**Reply by raybellwaves** [reply]
https://phausamann.github.io/sklearn-xarray/

**Comment by raybellwaves** [comment]
https://xcast-lib.github.io/
